lib/build.rs

```diff
@@ -13,11 +13,5 @@
 // limitations under the License.
 
 fn main() -> std::io::Result<()> {
-    println!("cargo:rerun-if-changed=build.rs");
-
-    if let Some(true) = version_check::supports_feature("map_first_last") {
-        println!("cargo:rustc-cfg=feature=\"map_first_last\"");
-    }
-
-    Ok(())
+    panic!("REPLACED BY THE `jj-lib` CRATE");
```

Comment:
Perhaps we should have a more detailed message here:

> This crate has been replaced by the `jj-lib` crate.

> Please upgrade to `jj-lib` version 0.8+. You can also downgrade to `jujutsu` version 0.7 if you need it to compile.

Reply:
I would also put the same message above the crate's README. Something like:

```
# WARNING

(Message)

# Original README follows
```

Reply:
I would also put the same message above the crate's README (for both crates). Something like:

```
# WARNING

(Message)

-------------

# Original README follows
```

Reply:
> Should we also have a similar message for jj-cli and jujutsu?

I did that at first, but I don't think it really matters because the lib crate gets compiled first, so the CLI crate is never even attempted.

I've addressed the other suggestions.

Reply:
> I don't think it really matters because the lib crate gets compiled first, so the CLI crate is never even attempted.

It's entirely up to you and doesn't really matter either way, but I would still do that, if only to make it easier for people to find the `build.rs` file that messes up their compilation. 

But again, it's fine as is.

Reply:
Sure, can't hurt.